pandas/tests/series/test_operators.py

```diff
@@ -1400,25 +1386,51 @@ def test_datetime_series_with_timedelta(self):
         assert_series_equal(result, expected)
         assert_series_equal(result2, expected)
 
-    def test_datetime_series_with_DateOffset(self):
+    def test_dt64_series_add_tick_DateOffset(self):
         # GH 4532
         # operate with pd.offsets
-        s = Series([Timestamp('20130101 9:01'), Timestamp('20130101 9:02')])
-
-        result = s + pd.offsets.Second(5)
-        result2 = pd.offsets.Second(5) + s
+        ser = Series([Timestamp('20130101 9:01'), Timestamp('20130101 9:02')])
         expected = Series([Timestamp('20130101 9:01:05'),
                            Timestamp('20130101 9:02:05')])
+
+        result = ser + pd.offsets.Second(5)
         assert_series_equal(result, expected)
+
+        result2 = pd.offsets.Second(5) + ser
         assert_series_equal(result2, expected)
 
-        result = s - pd.offsets.Second(5)
-        result2 = -pd.offsets.Second(5) + s
+    def test_dt64_series_sub_tick_DateOffset(self):
+        # GH 4532
+        # operate with pd.offsets
+        ser = Series([Timestamp('20130101 9:01'), Timestamp('20130101 9:02')])
         expected = Series([Timestamp('20130101 9:00:55'),
                            Timestamp('20130101 9:01:55')])
+
+        result = ser - pd.offsets.Second(5)
         assert_series_equal(result, expected)
+
+        result2 = -pd.offsets.Second(5) + ser
         assert_series_equal(result2, expected)
 
+        with pytest.raises(TypeError):
+            pd.offsets.Second(5) - ser
+
+    @pytest.mark.parametrize('cls_name', ['Day', 'Hour', 'Minute', 'Second',
```

Comment:
follow up to make this a fixture (to use above as well), prob could move the fixtures in tests/tseries/offsets to the top-level conftest